components/mailchimp/actions/create-campaign/create-campaign.mjs

```diff
@@ -46,147 +49,176 @@ export default {
         "all",
       ],
     },
-    segment_conditions: {
+    segmentConditions: {
+      label: "Segment conditions",
       type: "any",
       description: "Segment match conditions.",
       optional: true,
     },
-    subject_line: {
+    subjectLine: {
+      label: "Subject line",
       type: "string",
       description: "The subject line for the campaign.",
       optional: true,
     },
-    preview_text: {
+    previewText: {
+      label: "Preview text",
       type: "string",
       description: "The preview text for the campaign.",
       optional: true,
     },
     title: {
+      label: "Title",
       type: "string",
       description: "The title of the campaign.",
       optional: true,
     },
-    from_name: {
+    fromName: {
+      label: "From name",
       type: "string",
       description: "The 'from' name on the campaign (not an email address).",
       optional: true,
     },
-    reply_to: {
+    replyTo: {
+      label: "Reply to",
       type: "string",
       description: "The reply-to email address for the campaign. Note: while this field is not required for campaign creation, it is required for sending.",
       optional: true,
     },
-    use_conversation: {
+    useConversation: {
+      label: "Use conversation",
       type: "boolean",
       description: "Use Mailchimp Conversation feature to manage out-of-office replies.",
       optional: true,
     },
-    to_name: {
+    toName: {
+      label: "To name",
       type: "string",
       description: "The campaign's custom 'To' name.",
```

Comment:
Why this `To` is between single quotes?

Reply:
I actually did not work on this.
You reviewed the part I worked on yesterday. I had to do refactoring for the others which you are reviewing now